vendor/k8s.io/klog/OWNERS

```diff
@@ -1,4 +1,4 @@
-# See the OWNERS docs: https://git.k8s.io/community/contributors/guide/owners.md
+# See the OWNERS docs at https://go.k8s.io/owners
```

Comment:
Just for anyone else how is reviewing this...the `-1` line change in the diff occurs because of this line. 

(explicitly writing the comment because it's hard to find this in the huge diff)

Reply:
Oh this is a vendor file... :/

@rlenferink This change will need to go to the kubernetes/klog repo and then k8s.io/klog needs to be bumped here.
/lgtm cancel

Reply:
Ah, I completely missed that one. I’ll remove all the changes to the `vendor/` directory from this PR and open a PR for the other k8s repos.

Thanks for catching this. 

---
I’m still at FOSDEM this weekend so I’ll make the update tomorrow.

Reply:
> I’m still at FOSDEM this weekend so I’ll make the update tomorrow.

Sure. Enjoy FOSDEM! :)